tasks/publish.yaml

```
@@ -0,0 +1,28 @@
+includes:
+  - publish: https://raw.githubusercontent.com/defenseunicorns/uds-common-tasks/v0.1.2/tasks/publish.yaml
+
+variables:
+  - name: TARGET_REPO
+    default: oci://ghcr.io/defenseunicorns/packages/uds
+
+  - name: VERSION
+    description: The version of the package to build
+    # x-release-please-start-version
+    default: "1.10.1-uds.0"
+    # x-release-please-end
+  
+  - name: FLAVOR
+    default: "registry1"
+
+tasks:
+  - name: package
+    description: Build and publish the packages
+    actions:
+      - description: Create the packages
+        cmd: |
+          set -e
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml postgres-operator --no-progress --set FLAVOR=${FLAVOR}
+          ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml postgres-operator --no-progress --set FLAVOR=${FLAVOR}
```

```suggestion
          uds run -a amd64 -f tasks/create.yaml postgres-operator --no-progress --set FLAVOR=${FLAVOR}
          uds run -a arm64 -f tasks/create.yaml postgres-operator --no-progress --set FLAVOR=${FLAVOR}
```
Can we do this instead?

---

This is how we're doing it everywhere else currrently:
https://github.com/defenseunicorns/uds-core/blob/main/tasks/publish.yaml
https://github.com/defenseunicorns/uds-package-gitlab/blob/42e1f96de45381d514ff475715a3f7d301c24c37/tasks/publish.yaml

My aim was for consistency and I would prefer not to diverge unless we can document how we want to do this across uds repos and change it across them